lib/pluginUtils.js

```diff
@@ -134,7 +135,14 @@ function registerPluginName (fn) {
   this[kRegisteredPlugins].push(name)
 }
 
+function checkPluginHealthiness (fn) {
```

Comment:
Let's just check this gets in sync after #5139 gets merged into `main` 👍

Reply:
Yep, I'm marking the PR as draft